### PR TITLE
add Coinweb

### PR DESCRIPTION
**Pull request by TheRealSatar**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Coinweb

##### Twitter Link:
https://x.com/CoinwebOfficial

##### Website Link:
https://coinweb.finance/

##### Logo (High resolution, will be shown with rounded borders):
coinweb

##### Chain:
Ethereum

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
coinweb

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
12089

##### Short Description (to be shown on DefiLlama):
Coinweb is a Layer-2 protocol for building chain agnostic decentralized applications. Unchain the potential of your dApp with unlimited possibilities.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Services

##### methodology (what is being counted as tvl, how is tvl being calculated):
Currently, we calculate TVL as the staking of $CWEB tokens.

### CODE REVIEW

**Comment by llamabutler** [comment]
The adapter at projects/coinweb exports TVL: 
        
 
 
    ethereum-staking          14.77 M
    staking                   14.77 M
    ethereum                  0
    
    total                    0

File: projects/coinweb/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { staking } = require('../helper/staking');
+
+module.exports = {
+  ethereum: {
+    tvl: () => ({}),
+    staking: staking(
+      '0x13Fe7160858F2A16b8e4429DFf26c8a3A4b12b1B',
```
Comment (g1nt0ki) [inline projects/coinweb/index.js:7]: this is not a staking contract? 
image

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

File: projects/coinweb/index.js (hunk 1 above)
Comment (TheRealSatar) [reply projects/coinweb/index.js:7]: Hello @g1nt0ki , vesting contract.

Comment (g1nt0ki) [reply projects/coinweb/index.js:7]: sorry, we dont track vesting atm.

**Pull request closed without merging**